### ISSUE TITLE
Some issues during the installtion

### ISSUE THREAD

**Issue by smogol66** [JWahle/piCoreCDSP#2]
Hi,
I encounter some issues during the installation:
1. at line 94, I got a permission denied overwriting asound.conf, I had to change the owner of this file to run the script...
2. line 144 the utility pip did not install python3.8-pip
3. line 182; permission issues, had to change owner of the folder also
4. line 201 isn't already done?
last: having to restart the installlation several times, you must check if camilla is installed at the beginning, else you have to remove the folder manualli before starting the installation.
Else, it worked!
Tanks for your effort
Greatings!

**Comment by JWahle** [comment]
Hi,
What piCorePlayer version and Raspi are you using?
Did you use a fresh piCorePlayer installation or did you tinker with it beforehand?
Did you reboot the pi before executing the script again after failure?
